app/views/layouts/_header.html.erb

```diff
@@ -3,15 +3,48 @@
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
 <span class="navbar-toggler-icon"></span>
 </button>
+
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
```

Comment:
This file has the same problems as commented in _header2.html.erb

In addition, I'm not clear why we have two headers. If they're necessary, we should give them more descriptive names. If we don't need two, we should refactor to delete one and make sure we only use the other.